features/ticketing/tsrm-integration/src/main/java/org/opennms/netmgt/ticketer/tsrm/TsrmTicketerPlugin.java

```diff
@@ -231,6 +229,9 @@ public Ticket get(String ticketId) throws PluginException {
                 if (ownerGroup != null) {
                     ticket.addAttribute(OWNER_GROUP, ownerGroup.getValue());
                 }
+                if (reportedBy != null) {
+                    ticket.setUser(reportedBy.getValue());
+                }
```

Comment:
This change doesn't fit to me to solve the problem described in the issue. See here: http://issues.opennms.org/browse/NMS-8520.